_i18n/ru/_posts/2021-09-15-rest-api.md

```diff
@@ -14,7 +14,7 @@ REST API это HTTP API (иногда говорят WEB API) построен
 
 Прежде чем обсуждать конкретно, что из себя представляют принципы REST, нужно немного поговорить о его истории. Глобально, REST это не про то как делать HTTP API. REST это, то по каким принципам должен был быть построен весь WEB.
 
-> Since 1994, the REST architectural style has been used to guide the design and development of the architecture for the modern Web. This work was done in conjunction with my authoring of the Internet standards for the Hypertext Transfer Protocol (HTTP) and Uniform Resource Identifiers (URI), the two specifications that define the generic interface used by all component interactions on the Web.<br><br>— Roy Fielding, Architectural Styles and the Design of Network-based Software Architectures, p. 107.
+> Since 1994, the REST architectural style has been used to guide the design and development of the architecture for the modern Web. This work was done in conjunction with my authoring of the Internet standards for the Hypertext Transfer Protocol (HTTP) and Uniform Resource Identifiers (URI), the two specifications that define the generic interface used by all component interactions on the Web.<br /><br />— Roy Fielding, Architectural Styles and the Design of Network-based Software Architectures, p. 107.
```

Comment:
Может, эти `<br>` вообще  убрать? Два `<br>` — это же всё равно как параграф выглядит. Не настаиваю, просто предлагаю.
```suggestion
> Since 1994, the REST architectural style has been used to guide the design and development of the architecture for the modern Web. This work was done in conjunction with my authoring of the Internet standards for the Hypertext Transfer Protocol (HTTP) and Uniform Resource Identifiers (URI), the two specifications that define the generic interface used by all component interactions on the Web.
>
> — Roy Fielding, Architectural Styles and the Design of Network-based Software Architectures, p. 107.
```

Reply:
Да, согласен. Поправил